xbmc/epg/GUIEPGGridContainer.cpp

```diff
@@ -114,11 +114,11 @@ CGUIEPGGridContainer::CGUIEPGGridContainer(const CGUIEPGGridContainer &other)
   m_focusedProgrammeLayouts = other.m_focusedProgrammeLayouts;
   m_programmeLayouts        = other.m_programmeLayouts;
   m_rulerLayouts            = other.m_rulerLayouts;
-  m_channelLayout           = other.m_channelLayout;
-  m_focusedChannelLayout    = other.m_focusedChannelLayout;
-  m_programmeLayout         = other.m_programmeLayout;
-  m_focusedProgrammeLayout  = other.m_focusedProgrammeLayout;
-  m_rulerLayout             = other.m_rulerLayout;
+  m_channelLayout           = NULL;
+  m_focusedChannelLayout    = NULL;
+  m_programmeLayout         = NULL;
+  m_focusedProgrammeLayout  = NULL;
+  m_rulerLayout             = NULL;
```

Comment:
I wish I could delete it, but it's needed for Clone(). Clone() must be implemented (even though it's never called), which is an unfortunate design decision in the UI library.
Since deleting it is not an option, making it not crash certainly is not an improvement - should somebody ever call it some day.